diff -urN gmerlin-1.2.0/icons/Makefile.am gmerlin-1.2.0-patched/icons/Makefile.am
--- gmerlin-1.2.0/icons/Makefile.am	2012-06-08 22:50:36.000000000 +1100
+++ gmerlin-1.2.0-patched/icons/Makefile.am	2014-02-26 01:13:57.571338393 +1100
@@ -109,14 +109,14 @@
 
 install-data-hook:
 	test -z $(DESTDIR)$(prefix)/share/icons/hicolor/48x48/apps || $(mkdir_p) $(DESTDIR)$(prefix)/share/icons/hicolor/48x48/apps
-	ln -sf $(DESTDIR)$(pkgdatadir)/mixer_icon.png $(DESTDIR)$(prefix)/share/icons/hicolor/48x48/apps/gmerlin-alsamixer.png
-	ln -sf $(DESTDIR)$(pkgdatadir)/player_icon.png $(DESTDIR)$(prefix)/share/icons/hicolor/48x48/apps/gmerlin-player.png
-	ln -sf $(DESTDIR)$(pkgdatadir)/transcoder_icon.png $(DESTDIR)$(prefix)/share/icons/hicolor/48x48/apps/gmerlin-transcoder.png
-	ln -sf $(DESTDIR)$(pkgdatadir)/camelot_icon.png $(DESTDIR)$(prefix)/share/icons/hicolor/48x48/apps/gmerlin-camelot.png
-	ln -sf $(DESTDIR)$(pkgdatadir)/kbd_icon.png $(DESTDIR)$(prefix)/share/icons/hicolor/48x48/apps/gmerlin-kbd.png
-	ln -sf $(DESTDIR)$(pkgdatadir)/plugincfg_icon.png $(DESTDIR)$(prefix)/share/icons/hicolor/48x48/apps/gmerlin-plugincfg.png
-	ln -sf $(DESTDIR)$(pkgdatadir)/recorder_icon.png $(DESTDIR)$(prefix)/share/icons/hicolor/48x48/apps/gmerlin-recorder.png
-	ln -sf $(DESTDIR)$(pkgdatadir)/visualizer_icon.png $(DESTDIR)$(prefix)/share/icons/hicolor/48x48/apps/gmerlin-visualizer.png
+	ln -sf $(pkgdatadir)/mixer_icon.png $(DESTDIR)$(prefix)/share/icons/hicolor/48x48/apps/gmerlin-alsamixer.png
+	ln -sf $(pkgdatadir)/player_icon.png $(DESTDIR)$(prefix)/share/icons/hicolor/48x48/apps/gmerlin-player.png
+	ln -sf $(pkgdatadir)/transcoder_icon.png $(DESTDIR)$(prefix)/share/icons/hicolor/48x48/apps/gmerlin-transcoder.png
+	ln -sf $(pkgdatadir)/camelot_icon.png $(DESTDIR)$(prefix)/share/icons/hicolor/48x48/apps/gmerlin-camelot.png
+	ln -sf $(pkgdatadir)/kbd_icon.png $(DESTDIR)$(prefix)/share/icons/hicolor/48x48/apps/gmerlin-kbd.png
+	ln -sf $(pkgdatadir)/plugincfg_icon.png $(DESTDIR)$(prefix)/share/icons/hicolor/48x48/apps/gmerlin-plugincfg.png
+	ln -sf $(pkgdatadir)/recorder_icon.png $(DESTDIR)$(prefix)/share/icons/hicolor/48x48/apps/gmerlin-recorder.png
+	ln -sf $(pkgdatadir)/visualizer_icon.png $(DESTDIR)$(prefix)/share/icons/hicolor/48x48/apps/gmerlin-visualizer.png
 
 uninstall-local:
 	-rm -rf $(DESTDIR)$(prefix)/share/icons/hicolor/48x48/apps/gmerlin-alsamixer.png
